src/identities.rs

```diff
@@ -307,7 +303,8 @@ impl UserIdentity {
     /// Such a violation should be reported to the local user by the
     /// application, and resolved by
     ///
-    /// - Verifying the new identity with [`UserIdentity::request_verification`]
+    /// - Verifying the new identity with {@link
+    ///   UserIdentity::request_verification`]
```

Comment:
```suggestion
    ///   request_verification}
```

(which should make it fit on the previous line)